libs/isograph-babel-plugin/compileTag.js

```diff
@@ -1,42 +1,40 @@
-'use strict';
+"use strict";
 
-const pathModule = require('path');
+const pathModule = require("path");
 
 function compileTag(t, path, config) {
-  const tag = path.get('tag');
-
-  if (tag.isIdentifier({ name: 'iso' })) {
+  const callee = path.node.callee;
+  if (t.isIdentifier(callee) && callee.name === "iso" && path.node.arguments) {
     const { keyword, type, field } = getTypeAndField(path);
-    if (keyword === 'entrypoint') {
+    if (keyword === "entrypoint") {
       // This throws if the tag is invalid
-      compileImportStatement(t, path, type, field, 'entrypoint', config);
-    } else if (keyword === 'field') {
+      compileImportStatement(t, path, type, field, "entrypoint", config);
+    } else if (keyword === "field") {
       // No-op
       return false;
     } else {
       throw new Error(
-        "Invalid iso tag usage. Expected 'entrypoint' or 'field'.",
+        "Invalid iso tag usage. Expected 'entrypoint' or 'field'."
       );
     }
   }
-
   return false;
 }
 
 const typeAndFieldRegex = new RegExp(
-  '\\s*(entrypoint|field)\\s*([^\\.\\s]+)\\.([^\\s\\(]+)',
-  'm',
+  "\\s*(entrypoint|field)\\s*([^\\.\\s]+)\\.([^\\s\\(]+)",
+  "m"
 );
 
 function getTypeAndField(path) {
-  const quasis = path.node.quasi.quasis;
+  const quasis = path.node.arguments[0].quasis;
```

Comment:
This will probably throw if `iso()` is called by itself, i.e. if there is no argument

Reply:
This line here should protect from those problems

Reply:
What if arguments exists, but is of length 0?

Reply:
I was so sure that the evaluation rules of javascript would treat an array of length 0 as falsy, but I just checked and I was horribly wrong :)

Reply:
`iso()` is always a mistake, yeah. We should throw an error here.